eslint.config.mjs

```
@@ -1,7 +1,7 @@
 // For more info, see https://github.com/storybookjs/eslint-plugin-storybook#configuration-flat-config-format
 import storybook from "eslint-plugin-storybook";
 
-import { DT_USAGE_ESLINT_IMPORT_PATTERNS } from "./scripts/design-system/dt-usage-rules.mjs";
+import { DT_USAGE_ESLINT_FILE_GLOBS, DT_USAGE_ESLINT_IGNORES, DT_USAGE_ESLINT_IMPORT_PATTERNS } from "./scripts/design-system/dt-usage-rules.mjs";
```

ESLint

[error] Replace `·DT_USAGE_ESLINT_FILE_GLOBS,·DT_USAGE_ESLINT_IGNORES,·DT_USAGE_ESLINT_IMPORT_PATTERNS·` with `⏎··DT_USAGE_ESLINT_FILE_GLOBS,⏎··DT_USAGE_ESLINT_IGNORES,⏎··DT_USAGE_ESLINT_IMPORT_PATTERNS,⏎`

(prettier/prettier)